apps/app-article/src/helper.js

```diff
@@ -1,11 +1,16 @@
 export const isUserLoggedIn = () => {
-    return !(localStorage.getItem("uuid") === null && localStorage.getItem("token") === null);
+    return !(getCookie('token') === null);
 };
 
 export const shareArticle = (title, url, description) => {
     return AndroidNativeShare(title, url, description);
 };
 
+export const getCookie = (name) => {
+    let v = document.cookie.match('(^|;) ?' + name + '=([^;]*)(;|$)');
```

Comment:
This is a bit hard to follow, what's going on here? 🙂 Why do we need three capture groups?